scss/mixins.scss

```diff
@@ -60,6 +61,8 @@ $_o-brand-define-readme: "https://github.com/Financial-Times/o-brand/#obranddefi
     }
     // If from default component variables or a pre-defined variant.
     @if type-of($from) == 'string' or type-of($from) == 'null' {
+        // A brand must be configured before o-brand can be used.
+        $o-brand: oBrandGetCurrentBrand();
```

Comment:
A different variable name here would be great to avoid confusion the the global `$o-brand` variable.